### DISCUSSION TITLE
Help with Update

### DISCUSSION THREAD

**Discussion by styyokuda** [idaholab/moose#17918] [Q&A Getting Started] [answered]
Dear MOOSE Group,
I am trying to update and got an error message as:

% conda activate moose
% conda update --all
Collecting package metadata (current_repodata.json): failed

CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/idaholab/osx-64/current_repodata.json>
Elapsed: -

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
'https://conda.anaconda.org/idaholab/osx-64'

Please help.
Thomas

**Comment by milljm** [comment, marked as answer]
As the error message states, these errors are intermittent. I am positive if you try again, it will work. Normally, you can try again immediately after seeing an error like this, and it will work.

**Reply by styyokuda** [reply]
It is still not working. If it is not intermittent, what would you recommend me to do?
Please let me know.
Thomas

**Reply by milljm** [reply]
I wonder if Anaconda broke our stuff? The following URL does not work:
https://conda.anaconda.org/idaholab/osx-64/current_repodata.json

but all the packages are available:
https://conda.anaconda.org/idaholab/osx-64

We don't really have access to build the current_repodata.json file. Anaconda does this for you when you upload a package. Very strange. I'll look into it.

**Reply by rks171** [reply]
I had this same issue and what worked for me was to disable ssl verirification by typing the following and then trying again.

```
conda config --set ssl_verify False
```
